Runtime/Model/DataProvider/IMachineIdentifierDataProvider.cs

```diff
@@ -0,0 +1,8 @@
+﻿[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Backtrace.Unity.Tests.Runtime")]
+namespace Backtrace.Unity.Model.DataProvider
+{
+    internal interface IMachineIdentifierDataProvider
```

Comment:
This only provides the machine identifier. Why not name this `IMachineIdentifierProvider`?

Reply:
done